contribute/process/release-management.md

```diff
@@ -27,15 +27,14 @@ Release branches need to be created for the following repositories:
 - openebs/libcstor
 - openebs/cstor
 - openebs/istgt
-- openebs/external-storage
```

Comment:
Can we add these under manual tagging section. Probably with a new message saying, the following repos are being deprecated and will be manually tagged.

Reply:
```
openbs/maya
openebs/openebs-k8s-provisioner
```

Reply:
Done.